seeders/20221018204029-seeding-post.js

```diff
@@ -1,16 +1,121 @@
-const posts = require("./posts.json");
-const users = require("./users.json");
-const comments = require("./comments.json");
+// const posts = require("./posts.json");
+const { faker } = require("@faker-js/faker");
+// const comments = require("./comments.json");
+const User = require("../models/index").users;
+
+const users = [];
+const posts = [];
+const comments = [];
+
+async function createUser() {
+  for (let i = 1; i <= 1000; i++) {
+    const user = {
+      email: faker.internet.email(),
+      phone: faker.phone.number("###-###-####"),
+      username: faker.internet.userName(),
+      password: await User.encryptPassword(faker.internet.password()),
+      role: "user",
+      last_edit_date: faker.date.between(
```

Comment:
lets remove the last edit date, create a migration file to remove that as it is not going to be used.